docs/typography/word-break.md

```diff
@@ -0,0 +1,25 @@
+# Word Break
+
+ZSS contains utility classes for setting values for the `word-break` property.
+
+## Source files
+
+- [utilities/_word-break.scss](../../src/utilities/_word-break.scss)
+
+## Variables
+
+Since the word break CSS properties have a fixed set of values, there is no variable to configure.   
+
+## Generated CSS
+
+There is no generated CSS (based on variables).
```

Comment:
Hier komt dan toch wel wat uit?

`.u-break--word { word-break: break-word; }
.u-break--all { word-break: break-all; }`

Reply:
Geen generated content. Bij veel definities zie je dat er aan de hand van een variabele iets gegenereerd wordt (colors bijv. of voor verschillende break points).
Dit is hier niet het geval (er wordt helemaal niks gegenereerd)

Reply:
Toch maar even toegevoegd aangezien het wel gebruikelijk lijkt in de andere docs